### PR TITLE
Updated to 2024 and commented out old code for 2023 season.

### PR DESCRIPTION
**Pull request by SamueljWilson**

### CODE REVIEW

File: src/main/java/frc/robot/commands/DriveTrajectory.java [hunk 1]
```diff
@@ -24,14 +24,14 @@
 import edu.wpi.first.math.trajectory.Trajectory.State;
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.Command;
```
Comment (keira1008) [inline src/main/java/frc/robot/commands/DriveTrajectory.java]: This is a duplicate import.

File: src/main/java/frc/robot/commands/MoveArmSync.java [hunk 2]
```diff
@@ -5,12 +5,12 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.Command;
```
Comment (keira1008) [inline src/main/java/frc/robot/commands/MoveArmSync.java]: Here too.

**Review by keira1008 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/DriveTrajectory.java (hunk 1 above)
Comment (SamueljWilson) [reply src/main/java/frc/robot/commands/DriveTrajectory.java]: Fixed

File: src/main/java/frc/robot/commands/MoveArmSync.java (hunk 2 above)
Comment (SamueljWilson) [reply src/main/java/frc/robot/commands/MoveArmSync.java]: Fixed

**Review by keira1008 — APPROVED** [review]

**Pull request merged**